examples/netweaver_svpc_simple_example/main.tf

```diff
@@ -13,40 +13,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-resource "random_id" "random_suffix" {
-  byte_length = 4
-}
-
-locals {
-  gcs_bucket_name        = "post-deployment-bucket-${random_id.random_suffix.hex}"
-  gcs_bucket_static_name = "${var.nw_deployment_bucket}"
-}
-
-
-resource "google_storage_bucket" "deployment_bucket" {
-  name          = "${local.gcs_bucket_name}"
-  force_destroy = true
-  location      = "${var.region}"
-  storage_class = "REGIONAL"
-  project       = "${var.project_id}"
-}
 
-data "template_file" "post_deployment_script" {
-  template = "${file("${path.module}/files/templates/post_deployment_script.tpl")}"
+provider "google" {
+  version     = "~> 2.6.0"
+  credentials = "${file("credentials.json")}"
 }
 
-data "template_file" "netweaver" {
-  template = "${file("${path.module}/files/netweaver_startup.sh")}"
-}
-
-resource "google_storage_bucket_object" "post_deployment_script" {
-  name    = "post_deployment_script.sh"
-  content = "${data.template_file.post_deployment_script.rendered}"
-  bucket  = "${google_storage_bucket.deployment_bucket.name}"
-}
-
-module "example" {
-  source                 = "../../../examples/netweaver_simple_example"
+module "gcp_netweaver_svpc" {
+  source                 = "../../modules/netweaver_svpc"
+  post_deployment_script = "${var.post_deployment_script}"
```

Comment:
do you want me to change the naming convention in the modules folder declaration : eg instead of netweaver_simple_example , name it as netweaver , then do it for all the respective modules.